xbmc/windowing/gbm/DRMUtils.cpp

```diff
@@ -109,10 +109,13 @@ drm_fb * CDRMUtils::DrmFbGetFromBo(struct gbm_bo *bo)
   memset(offsets, 0, 16);
 #endif
 
-  if (modifiers[0] == DRM_FORMAT_MOD_INVALID)
-    modifiers[0] = DRM_FORMAT_MOD_LINEAR;
+  uint32_t flags = 0;
 
-  CLog::Log(LOGDEBUG, "CDRMUtils::%s - using modifier: %lli", __FUNCTION__, modifiers[0]);
+  if (modifiers[0] > DRM_FORMAT_MOD_LINEAR && modifiers[0] != DRM_FORMAT_MOD_INVALID)
```

Comment:
Since `modifiers` is unsigned and `DRM_FORMAT_MOD_LINEAR` is `0` using one of the suggestions below could make the code shorter or clearer.
```suggestion
  if (modifiers[0] && modifiers[0] != DRM_FORMAT_MOD_INVALID)
```
or
```suggestion
  if (modifiers[0] != DRM_FORMAT_MOD_LINEAR && modifiers[0] != DRM_FORMAT_MOD_INVALID)
```